scripts/homework_scanner.py

```
@@ -0,0 +1,353 @@
+        RuntimeError: If gh CLI call fails.
+    """
+    review_comments = _gh_api(
+        f"repos/{owner}/{repo}/pulls/{pr_number}/comments"
+    )
+    reviews = _gh_api(f"repos/{owner}/{repo}/pulls/{pr_number}/reviews")
+    return review_comments, reviews
+
+
+def _gh_api(endpoint: str) -> list[dict[str, Any]]:
+    """Call gh api and return parsed JSON list."""
+    result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)